### PR TITLE
Update README.rst -> Hint concerning the import of fancycompleter in .pdbrc.py

### PR DESCRIPTION
**Pull request by GHPS**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -257,10 +257,14 @@ default value:
...
 ``line_number_color = Color.turquoise``
-  The color to use for line numbers.
+  The color to use for line numbers. 
+  
+  (Note: Color.* are taken from fancycompleter so remember to import this module first.)
 
 ``filename_color = Color.yellow``
   The color to use for file names when printing the stack entries.
+  
+  (Note: Color.* are taken from fancycompleter so remember to import this module first.)
```
Comment (blueyed) [inline README.rst:267]: Thanks.  Makes sense in general I think.
Note however that you can also use strings there directly, and even more (see `current_line_color = "39;49;7"` below (but have not really tested it with `line_number_color` / `filename_color`.

Comment (GHPS) [reply README.rst:267]: So the hint could be:
(Note: Color.* are taken from fancycompleter so remember to import this module first
or resort to direct formatting like "39;49;7".)

**Comment by blueyed** [comment]
Thanks for suggesting this, I've revisited the README in this regard.
Please check if it makes sense.

**Comment by GHPS** [comment]
The README is much better now!

Currently missing is the anchor for the "ANSI_escape_code_note" therefore the
jump to this part of the text doesn't work.

Interesting that the import of "fancycompleter" isn't necessary at all and can
be done via pdb...

Meanwhile I tested remote debugging with pdb++ (using remote-pdb).
This is a great feature for debugging programs that produce output on
the standard display. One can telnet into the debugging terminal from
a different shell session and still see the undisturbed output of the
program. 

In my tests remote-pdb and pdb++ worked perfectly together. 
I therefore suggest mentioning this feature in the README as well.

**Comment by blueyed** [comment]
Thanks, fixed in https://github.com/pdbpp/pdbpp/commit/351f69476d9390352de1fb5feb1d7a1d716baaa4.

> In my tests remote-pdb and pdb++ worked perfectly together.
> I therefore suggest mentioning this feature in the README as well.

Yes, might make sense to have a "related projects" or similar section.  There are some other "enhanced terminal debuggers" also, which could be mentioned there then.

**Pull request closed without merging**